src/Nest/Search/Search/Highlighting/HighlightField.cs

```diff
@@ -110,7 +118,14 @@ public class HighlightFieldDescriptor<T> : DescriptorBase<HighlightFieldDescript
 
 		string IHighlightField.BoundaryChars { get; set; }
 
-		HighlighterType? IHighlightField.Type { get; set; }
+		HighlighterType? IHighlightField.Type
+		{
+#pragma warning disable CS0618 // Type or member is obsolete
+			get { return Self.CustomType.ToEnum<HighlighterType>(); }
```

Comment:
ah, ok :+1: Threw me off because the previous PR had a null check, but I hadn't looked at the ToEnum() implementation, just assumed.